src/modules/dashboard/components/ActionsPage/ActionsPageEvent/ActionsPageEvent.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import { ParsedEvent } from '~data/index';
+import TransactionMeta from '../TransactionMeta';
+import styles from './ActionsPageEvent.css';
+import UserPermissions from '~admin/Permissions/UserPermissions';
+import TextDecorator from '~lib/TextDecorator';
+import UserMention from '~core/UserMention';
+
+const displayName = 'dashboard.ActionsPage.ActionsPageEvent';
+
+const MSG = defineMessages({
+  eventTitle: {
+    id: 'dashboard.ActionsPage.ActionsPageEvent.eventTitle',
+    defaultMessage: `{from} paid {value} from {team} to {to}.`,
+  },
+});
+
+interface Props {
+  event: ParsedEvent;
+  transactionHash?: string | null;
+  createdAt?: number;
+}
+
+const ActionsPageEvent = ({ createdAt, transactionHash }: Props) => {
+  // Mocked roles - Please make me smarter
```

Comment:
When adding comments like this, please leave a `@TODO` doctag so we can easily find them later.

```suggestion
  // @TODO Mocked roles - Please make me smarter
```

or

```suggestion
  /*
   * @TODO Mocked roles - Please make me smarter
   */
```